salt/modules/upstart.py

```diff
@@ -97,7 +97,10 @@ def _find_utmp():
             result[os.stat(utmp).st_mtime] = utmp
         except Exception:
             pass
-    return result[sorted(result).pop()]
+    if len(result) > 0:
```

Comment:
you can just do 
```
if result:
```

Reply:
Fixed.

Reply:
@gtmanfred looks like this breaks Py3 support. On Ubuntu 18.04/bionic this currently results in:
```
TypeError: '>' not supported between instances of 'dict' and 'int'
```
Will create a PR to use `if len(result) > 0:` again.

Reply:
See #46293